packages/rancher-alerting-drivers/charts/values.yaml

```diff
@@ -0,0 +1,86 @@
+# Default values for rancher-alerting-driver.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+global:
+  cattle:
+    # the registry where all images will be pulled from
+    systemDefaultRegistry: ""
+  preInstallHook:
+    securityContext:
+      runAsNonRoot: true
+      runAsUser: 1000
+    image:
+      repository: rancher/rancher-agent
+      tag: v2.5.7 # TODO: change to v2.5.8
+      pullPolicy: IfNotPresent
+  # set this value if you want the sub-charts to be installed into
+  # a namespace rather than where this chart is installed
+  namespaceOverride: ""
+
+prom2teams:
+  enabled: false
+  nameOverride: "prom2teams"
+  fullnameOverride: ""
+  image:
+    repository: rancher/mirrored-idealista-prom2teams
+    tag: 3.2.1
+    pullPolicy: IfNotPresent
+  prom2teams:
+    connector:
+    connectors: {}
+    # group_alerts_by can be one of
+    # ("name" | "description" | "instance" | "severity" | "status" | "summary" | "fingerprint" | "runbook_url")
+    group_alerts_by:
+    # loglevel can be one of (DEBUG | INFO | WARNING | ERROR | CRITICAL)
+    loglevel: INFO
```

Comment:
@jiaqiluo  @cbron should we default to DEBUG or is it better to default to INFO and let a user modify to DEBUG?

Reply:
I'd say info